best-practice-secure-comms.md

```diff
@@ -342,6 +348,9 @@ Servers SHOULD NOT advertise multicast DNS-SD, except where a DNS server is not
   - As of version v1.2, AMWA IS-04 includes DNS-SD announcements of Node APIs.
     However, these may be deprecated and removed from later versions of the spec.
 
+Servers SHALL use a name listed in the Common Name and/or Subject Alternate Name fields of its certificate
+in DNS-SD advertisements whether unicast or multicast.
```

Comment:
So regarding mDNS announcements, the CN (described in the certificate) would be the (mdns) hostname?

Reply:
It should either be the CN or one of the Subject Alternate Names, yes